src/Matrix.jl

```diff
@@ -567,6 +567,7 @@ end
 
 Base.IteratorSize(::Type{<:MatrixElem}) = Base.HasShape{2}()
 
+Base.keys(M::MatElem) = CartesianIndices(axes(M))
```

Comment:
could you add a simple test exercising this?

Reply:
Done